### PR TITLE
Rename Camera Methods

### PR DESCRIPTION
**Pull request by clintpurser**
Rename camera methods to follow dart guidelines.

Also included driveby fixing warning about not have a return type on `deactivate()`

### CODE REVIEW

File: example/viam_example_app/lib/screens/stream.dart [hunk 1]
```diff
@@ -89,7 +89,7 @@ class _StreamScreenState extends State<StreamScreen> {
   }
 
   @override
-  deactivate() {
+  void deactivate() {
```
Comment (clintpurser) [inline example/viam_example_app/lib/screens/stream.dart:92]: driveby, this was giving a warning about not having a return type.

**Review by stuqdog — APPROVED** [review]

**Review by benjirewis — APPROVED** [review]

**Review by purplenicole730 — APPROVED** [review]
